src/shared/discover-image.c

```diff
@@ -63,6 +63,12 @@ static const char* const image_search_path[_IMAGE_CLASS_MAX] = {
                             "/var/lib/extensions\0"         /* the main place for images */
                             "/usr/local/lib/extensions\0"
                             "/usr/lib/extensions\0",
+
+        [IMAGE_CONFIGURATION] = "/etc/configuration\0"             /* only place symlinks here */
```

Comment:
hmm, the dir name sounds a bit unspecific. And for the other three image classes we use plural, i.e. "machines", "portables" and "extensions" all is plural.

Maybe call the dir just "syscfgs"? i.e. /etc/syscfgs/, /run/syscfgs/ and so on?